core/common/src/main/java/alluxio/conf/ConfigurationValueOptions.java

```diff
@@ -19,6 +19,8 @@
   private boolean mUseRawValue = false;
 
   /**
+   * Creates default configuration value options, preventing this class to be instanced directly.
```

Comment:
Actually what prevents the creation of a class directly from constructor is the private constructor. This method just provides an instance with defaults